molecule/default/tests/test_default_node.py

```diff
@@ -0,0 +1,16 @@
+import os
+import pytest
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('node')
+
+
+@pytest.mark.parametrize('pkg', [
+    'open-iscsi',
+    'nfs-common',
+])
+def test_pkg(host, pkg):
+    package = host.package(pkg)
+    assert package.is_installed
```

Comment:
Up until now, we have had neither scenario-specific test cases (all current test cases are in `molecule/resources/verify/from_outside`) nor any test cases using `testinfra`. Can you explain why you want to introduce both concepts for this case?

Reply:
In short, because I was using the following example and my molecule knowledge is pretty limited. Do you have a recommended alternative way to implement this?